updateParticipantData.json

```diff
@@ -34,6 +34,10 @@
 		"dataType": "number",
 		"mustExist": true
 	},
+    "state.253532712": {
+        "values": [723775357, 153444133, 572474909, 308427446, 211228524, 277568192, 611398522, 181769837, 178420302],
+        "dataType": "number"
+    },
```

Comment:
Hi @Gbarra9 the "mustExist" property in `updateParticipantData.json` means the property must already exist in the profile to allow it to be updated in the `updateParticipantData` API.

That means the value must have been pushed through the `submitParticipantsData` API initially, and exist in the profile to allow it to be updated. Not sure if you need that here, but I wanted to point out that extra variable you see for other items in the update.json file.

Madhuri will likely know what API is used for this.

`mustExist` usage (applies to the `update` API only, not the `submit` API): https://github.com/episphere/connectFaas/blob/ba0473d59789e008464a17d8c76eacb0f0c44995/utils/sites.js#L426